### PR TITLE
Update PR status

### PR DESCRIPTION
**Pull request by LALAYANG**

### CODE REVIEW

**Comment by LALAYANG** [comment]
For `InspiredAFix` tests, I added the commit link in which the developers fixed them, and the PR link which inspired developers to fix as a note, but the format checker failed.

File: pr-data.csv [hunk 1]
```diff
@@ -947,8 +947,8 @@ https://github.com/FasterXML/jackson-dataformats-binary,d78d13c574a4b139438841bd
...
-https://github.com/FasterXML/jackson-datatypes-collections,8b88868ac3012783fb7f1bda00309c823172e02a,guava,com.fasterxml.jackson.datatype.guava.FluentIterableTest.testSerialization,ID,,,
-https://github.com/FasterXML/jackson-datatypes-collections,8b88868ac3012783fb7f1bda00309c823172e02a,guava,com.fasterxml.jackson.datatype.guava.FluentIterableTest.testWrappedSerialization,ID,,,
+https://github.com/FasterXML/jackson-datatypes-collections,8b88868ac3012783fb7f1bda00309c823172e02a,guava,com.fasterxml.jackson.datatype.guava.FluentIterableTest.testSerialization,ID,InspiredAFix,https://github.com/FasterXML/jackson-datatypes-collections/commit/9c14eaa4fa6f3c57623e6cb6573fd604928165a0,https://github.com/FasterXML/jackson-datatypes-collections/pull/91
+https://github.com/FasterXML/jackson-datatypes-collections,8b88868ac3012783fb7f1bda00309c823172e02a,guava,com.fasterxml.jackson.datatype.guava.FluentIterableTest.testWrappedSerialization,ID,InspiredAFix,https://github.com/FasterXML/jackson-datatypes-collections/commit/9c14eaa4fa6f3c57623e6cb6573fd604928165a0,https://github.com/FasterXML/jackson-datatypes-collections/pull/91
```
Comment (winglam) [inline pr-data.csv:951]: @LALAYANG thanks for the changes. For these two lines, please have the PR link be the PR that inspired a fix and the commit be in the Notes column

Comment (LALAYANG) [reply pr-data.csv]: Hi Wing, I made the changes, but the checker still failed

Comment (winglam) [reply pr-data.csv]: To add notes, can you please follow the instructions in the README?
https://github.com/TestingResearchIllinois/idoft#adding-notes

Namely, create an issue report describing the fact that this commit link actually introduces a fix for these two flaky tests.

File: pr-data.csv [hunk 2]
```diff
@@ -947,8 +947,8 @@ https://github.com/FasterXML/jackson-dataformats-binary,d78d13c574a4b139438841bd
...
-https://github.com/FasterXML/jackson-datatypes-collections,8b88868ac3012783fb7f1bda00309c823172e02a,guava,com.fasterxml.jackson.datatype.guava.FluentIterableTest.testSerialization,ID,,,
-https://github.com/FasterXML/jackson-datatypes-collections,8b88868ac3012783fb7f1bda00309c823172e02a,guava,com.fasterxml.jackson.datatype.guava.FluentIterableTest.testWrappedSerialization,ID,,,
+https://github.com/FasterXML/jackson-datatypes-collections,8b88868ac3012783fb7f1bda00309c823172e02a,guava,com.fasterxml.jackson.datatype.guava.FluentIterableTest.testSerialization,ID,InspiredAFix,https://github.com/FasterXML/jackson-datatypes-collections/pull/91,https://github.com/TestingResearchIllinois/idoft/issues/48
+https://github.com/FasterXML/jackson-datatypes-collections,8b88868ac3012783fb7f1bda00309c823172e02a,guava,com.fasterxml.jackson.datatype.guava.FluentIterableTest.testWrappedSerialization,ID,InspiredAFix,https://github.com/FasterXML/jackson-datatypes-collections/pull/91,https://github.com/TestingResearchIllinois/idoft/issues/48
```
Comment (winglam) [inline pr-data.csv:951]: These two columns are missing a comma.
```InspiredAFix,https:``` -> ```InspiredAFix,,https:```

Comment (winglam) [reply pr-data.csv:951]: Please ignore the preceding comment. The problem with the failed build appears to be an issue with the checker.

**Comment by winglam** [comment]
I have resolved the problem with the format checker and updated your repo with the fixed format checker. The build now passes with the updated checker. Thanks for the changes!

**Pull request merged**